Makefile

```diff
@@ -23,16 +24,16 @@ check-swagger:
 modified:
 	git ls-files --modified | xargs git add
 
-update: update-blueprint update-catalog-deployment update-iaas update-content update-pipeline update-project
+update: update-blueprint update-catalog-deployment update-iaas update-content update-pipeline update-project update-cmx
 
 update-blueprint:
 	curl --insecure 'https://${SWAGGER_ENDPOINT}/blueprint/api/swagger/swagger-api-docs?group=2019-09-12' | python3 -mjson.tool > swagger/vra-blueprint.json
 
 update-catalog-deployment:
-	curl --insecure 'https://${SWAGGER_ENDPOINT}/deployment/api/swagger/swagger/v2/api-docs?group=2019-01-15' | python3 -mjson.tool > swagger/vra-catalog-deployment.json
+	curl --insecure 'https://${SWAGGER_ENDPOINT}/deployment/api/swagger/swagger/v2/api-docs?group=2020-08-25' | python3 -mjson.tool > swagger/vra-catalog-deployment.json
```

Comment:
Could you  share the documentation for checking this version?

Reply:
Go to any vRA 8.6.2 environment, and check the api-docs webpage, you will see the latest API version published. For reference, this is vRA Cloud: https://www.mgmt.cloud.vmware.com/deployment/api/swagger/swagger-ui.html?urls.primaryName=catalog